lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -367,5 +367,27 @@ def self.add_quotes(value)
         value.start_with?('"') && value.end_with?('"')
       value
     end # add_quotes
+
+    # extract value of property
```

Comment:
IMO these utility methods need more documentation.  It would be good to provide some examples that show how/why to use these for developers that are trying to understand the various methods in the library.